src/WorkBC.ElasticSearch.Search/Queries/JobSearchQuery.cs

```diff
@@ -146,9 +150,8 @@ public enum LocationDistanceKm
         /// <summary>
         ///     Build JSON string that will be used to query Elastic Search
         /// </summary>
-        public async Task<string> ToJson(IConfiguration configuration, string jsonFileName)
+        public async Task<string> ToJson(IConfiguration configuration, string json)
```

Comment:
Rather than pass in a JSON resource that must be part of either the executing assembly or the calling assembly, I think it's better to pass in the JSON string since this makes it easier to test.